qa/standalone/scrub/osd-scrub-snaps.sh

```diff
@@ -90,57 +90,58 @@ function create_scenario() {
     # Don't need to use ceph_objectstore_tool() function because osd stopped
 
     JSON="$(ceph-objectstore-tool --data-path $dir/${osd} --head --op list obj1)"
-    ceph-objectstore-tool --data-path $dir/${osd} "$JSON" --force remove
+    ceph-objectstore-tool --data-path $dir/${osd} "$JSON" --force remove || return 1
```

Comment:
Why do we want to tolerate an error in every one of these? Do we expect stale data we might need to clean up?

Reply:
Actually, this change causes the script to not tolerate errors.  Before I just assumed that these commands can not fail, so didn't bother to check.

Reply:
Oh dur.
👍